views/Footer.svelte

```diff
@@ -11,9 +11,11 @@
       <div class="s-q-item__footer__sources">
         {#if item.sources.length > 1}Quellen:{:else}Quelle:{/if}
         {#each item.sources as source, index}
-          {#if source.text !== ''}
+          {#if source.text !== ""}
             {#if source.link && source.link.url && source.link.isValid}
-              <a href={source.link.url} target="blank" rel="noopener noreferrer">{source.text}</a>{:else}{source.text}{/if}{#if index !== item.sources.length - 1 && item.sources[index + 1] !== ''},&nbsp;
+              <a href={source.link.url} target="blank" rel="noopener noreferrer"
+                >{source.text}</a
```

Comment:
Cosmetics: Is there a reason for the line break before the bracket `>` and also after the `</a`?

Reply:
I am using the svelte for VS Code extension. It allows to format `.svelte` components using prettier formatter. I will try to see if I can improve it.

Reply:
I was able to rearrange the tags a bit and the formatting is improved now: https://github.com/nzzdev/Q-choropleth/pull/62/commits/34046572fc36301e7ac8132b7ba608fc8adaffac